docs/static/releasenotes.asciidoc

```diff
@@ -57,6 +57,26 @@ This section summarizes the changes in the following releases:
 [[logstash-7-15-2]]
 === Logstash 7.15.2 Release Notes
 
+[[known-issue-7-15-2]]
+==== Known issue
+
+Release 7.15.2 includes filter versions that will not run on JDK 8.
+This issue is fixed in the latest versions: 
+
+* https://www.elastic.co/guide/en/logstash-versioned-plugins/current/v3.1.12-plugins-filters-date.html[date filter 3.1.12]
+* https://www.elastic.co/guide/en/logstash-versioned-plugins/current/v7.2.7-plugins-filters-geoip.html[geoip filter 7.2.7]
```

Comment:
Does the lockfile affect which versions can be manually updated? That is, can a user running 7.15.2 upgrade to these plugin versions, or do we need to suggest an upgrade to 7.16.1 as well.

Reply:
A user who downloads 7.15.2 could certainly install the updated versions of the plugins, in the usual way.

The lockfile is a snapshot of the plugins that are _currently_ installed, including all of their dependencies, and the plugin manager uses this info when installing or updating plugins to ensure that the dependency graph can be satisfied (and writes a new lockfile when it is done).

Reply:
Nice explanation, thank you!  I guess I've never thought about this before. Good to know. 😄